jsdojo/3-7/script.js

```diff
@@ -0,0 +1,9 @@
+let hp = 100;
+
+while (hp > 0) {
+
+    const attack = Math.floor(Math.random() * 30) + 1;
+    console.log('Damaged the monster with an attack value of ' + attack);
```

Comment:
FYI, you can also use [Template literals (Template strings)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) instead.